frontend/src/components/SyncEditor/SyncEditor.tsx

```diff
@@ -652,12 +653,22 @@ export function SyncEditor(props: SyncEditorProps): JSX.Element {
     if (variant === 'toolbar') {
       height -= 36
     }
+    if (editorHasErrors) {
+      height -= 36
+    }
     editorRef?.current?.layout({ width, height })
     setShowCondensed(width < 500)
   })
 
   return (
     <div ref={pageRef} className="sync-editor__container">
+      {editorHasErrors && (
+        <Alert
+          variant="danger"
+          title="Changes will be ignored until errors are fixed."
```

Comment:
- New string needs to be translated
- Text could be clearer. Would this be accurate: "Form edits will be ignored until YAML syntax errors are fixed"